pkg/cluster/util.go

```diff
@@ -227,13 +227,19 @@ func (c *Cluster) logServiceChanges(role PostgresRole, old, new *v1.Service, isU
 	}
 }
 
-func (c *Cluster) getPostgresContainer(podSpec *v1.PodSpec) v1.Container {
+func getPostgresContainer(podSpec *v1.PodSpec) v1.Container {
```

Comment:
you can make `pgContainer` a [named return value](https://tour.golang.org/basics/7)